webapp/src/widgets/buttons/button.tsx

```diff
@@ -5,6 +5,9 @@ import React from 'react'
 import './button.scss'
 import {Utils} from '../../utils'
 
+export type ButtonSize = 'xsmall' | 'small' | 'medium' | 'large'
+export type Emphasis = 'primary' | 'secondary' | 'tertiary' | 'quaternary' | 'gray' | 'danger' | 'default' | 'link'
```

Comment:
This is not needed for this PR and is parely an enhancement. By creating a type of allowed values, the IDE can directly show allowed properties in autocomplete suggestions.